### PR TITLE
add e2e test for timeline tool

### PR DESCRIPTION
**Pull request by qlj-lijuan**
- fix the issue that setStatsVar title is sometimes not called
- add unit test to make sure statsVar title is set properly
- add e2e test for timeline page

### CODE REVIEW

File: static/js/tools/__snapshots__/statsvar_menu.test.tsx.snap [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (shifucun) [inline static/js/tools/__snapshots__/statsvar_menu.test.tsx.snap:1]: can you add comments in the test file on how to update these snap files?

File: static/js/tools/timeline_page.test.tsx [hunk 2]
```diff
@@ -63,9 +64,59 @@ test("Single place and single stats var", () => {
...
+            Median_Age_Person: { md: "", mprop: "age", pt: "Person", pvs: {} },
+            Count_Person: { md: "", mprop: "count", pt: "Person", pvs: {} },
+          });
+          expect(wrapper.state().statsVarTitle).toEqual({
+            Median_Age_Person: "Median age",
+            Count_Person: "Population",
+          });
+          expect(
+            pretty(wrapper.find("#chart-region").getDOMNode().innerHTML)
+          ).toMatchSnapshot();
```
Comment (shifucun) [inline static/js/tools/timeline_page.test.tsx:88]: how does this test know which snapshot file to compare?

Comment (qlj-lijuan) [reply static/js/tools/timeline_page.test.tsx:88]: The snapshot file has the file name: TestFileName.snap. If multiple snapshots are created within one test, the snapshots would have the name "test name + an index ", for example "`Single place and single stats var 1`"

File: static/js/tools/__snapshots__/statsvar_menu.test.tsx.snap (hunk 1 above)
Comment (qlj-lijuan) [reply static/js/tools/__snapshots__/statsvar_menu.test.tsx.snap:1]: done.

File: static/js/tools/__snapshots__/statsvar_menu.test.tsx.snap [hunk 3]
```diff
@@ -1,4 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
+// update the snapshots with command: npm test -- -u
```
Comment (shifucun) [inline static/js/tools/__snapshots__/statsvar_menu.test.tsx.snap:2]: can you move this to the README file https://github.com/datacommonsorg/website/blob/master/README.md

Here might be override when creating a new snapshot.

Comment (qlj-lijuan) [reply static/js/tools/__snapshots__/statsvar_menu.test.tsx.snap:2]: Right. Done

**Pull request merged**